luminoso_api/json_stream.py

```diff
@@ -254,6 +260,13 @@ def _read_csv(reader, header, encode_fn):
         yield row_dict
 
 
+def convert_date(date_string, date_format):
+    if date_format != 'epoch':
+        return datetime.datetime.strptime(date_string, date_format).timestamp()
```

Comment:
And yes, it's not uncommon for us to get a file with dates that are slightly out of sync with each other.